plugins/mcreator-localization/help/default/block/hit_sound.md

```diff
@@ -0,0 +1,2 @@
+This is the sound played when the player mines the block.
+Note: This sound is a loop played until the block is broken.
```

Comment:
I would re-word this to say sound is played in loop while breaking until the block is broken